weather/templates/weather/partials/alert-summary.html

```diff
@@ -0,0 +1,13 @@
+<weathergov-alert-list>
+    <div class="margin-top-0 margin-bottom-2">
+        {% for alert in alerts %}
+            <div class="usa-alert usa-alert--warning usa-alert--slim wx-alert margin-top-0 margin-bottom-1 padding-y-05" role="alert" data-alert-level="{{ alert.metadata.level.text | lower }}">
```

Comment:
I'd kind of like to use React style line breaks and a 120 character line length. I think it would vastly improve the readability of the templates. Just a suggestion, though, not set on it.

What are we using for linting? I've liked black the times I've used it recently.

Reply:
To clarify that last comment, this would be something like:

```
<div
  class="usa-alert usa-alert--warning usa-alert--slim wx-alert margin-top-0 margin-bottom-1 padding-y-05"
  role="alert"
  data-alert-level="{{ alert.metadata.level.text | lower }}"
>
```

Reply:
Good call. As with your comment further below, we need to set up linting across the board. Ticket [here](https://vlab.noaa.gov/gitlab-licensed/NWS/Systems/DIS/Weather.gov-2.0/-/issues/105).